tests/ztest/base/src/main.c

```diff
@@ -154,7 +154,7 @@ static void rule_test_teardown(void *data)
 	 */
 	zassert_equal(fixture->state, RULE_STATE_AFTER_EACH, "Unexpected state");
 #ifdef CONFIG_ZTEST_SHUFFLE
```

Comment:
shouldn't it be for `CONFIG_ZTEST_REPEAT` now ?

Reply:
Agreed.

Reply:
done.